component/module/file/file_test.go

```diff
@@ -2,6 +2,7 @@ package file
 
 import (
```

Comment:
This test had to change a lot since it no longer had direct access to `flow`, I didnt want to remove it entirely so now it uses the http api to test for health.